server/spec/lib/data_import/my_anime_list/extractor/media_spec.rb

```diff
@@ -0,0 +1,374 @@
+        expect(subject.synopsis).to eq(expected_synopsis)
+      end
+    end
+
+    describe '#poster_image' do
+      it 'should return an image link' do
+        expect(subject.poster_image).to eq('http://cdn.myanimelist.net/images/manga/1/157931.jpg')
+      end
+    end
+
+    describe '#subtype' do
+      it 'should return a Symbol' do
+        expect(subject.subtype).to be_a(Symbol)
+      end
+
+      it 'should return manga' do
+        expect(subject.subtype).to eq(:manga)
+      end
+
+      it 'should return novel' do
+        subject = described_class.new({ type: 'Novel' }.to_json)
+        expect(subject.subtype).to eq(:novel)
+      end
+
+      it 'should return manuha' do
+        subject = described_class.new({ type: 'Manuha' }.to_json)
+        expect(subject.subtype).to eq(:manuha)
+      end
+
+      it 'should return oneshot' do
+        subject = described_class.new({ type: 'oneshot' }.to_json)
+        expect(subject.subtype).to eq(:oneshot)
+      end
+
+      it 'should return doujin' do
+        subject = described_class.new({ type: 'Doujin' }.to_json)
+        expect(subject.subtype).to eq(:doujin)
+      end
+    end
+
+    describe '#titles' do
+      it 'should return the Romaji title of media' do
+        expect(subject.titles[:en_jp]).to eq('Berserk')
+      end
+      it 'should return the English title of media' do
+        expect(subject.titles[:en_us]).to eq('Berserk')
+      end
+      it 'should return the Japanese title of media' do
+        expect(subject.titles[:ja_jp]).to eq('ベルセルク')
+      end
+      it 'should return nil if English or Japanese title does not exist' do
+        subject = described_class.new({ other_titles: {} }.to_json)
+
+        expect(subject.titles[:en_us]).to be_nil
+        expect(subject.titles[:ja_jp]).to be_nil
+      end
+    end
+
+    describe '#abbreviated_titles' do
+      it 'should return an array' do
+        expect(subject.abbreviated_titles).to be_an(Array)
+      end
+      it 'should return all synonyms titles' do
+        expect(subject.abbreviated_titles).to eq(['Berserk: The Prototype'])
+      end
+    end
+
+    describe '#genres' do
+      it 'should return an array' do
+        expect(subject.genres).to be_an(Array)
+      end
+
+      it 'should return all genres' do
+        expect(subject.genres).to eq(%w[Action Adventure Demons
```

Comment:
yeah it's wrong on `()`, I've checked the config.  This was 19 hours ago, probably from before I tweaked the configs or something idk